packages/tasit-sdk/src/TasitSdk.js

```diff
@@ -0,0 +1,4 @@
+import Account from "tasit-account";
+import { Contract } from "tasit-action";
```

Comment:
I wonder if it will be confusing at all to call this Contract when `tasit-contracts` will also have contracts. The contract here is more like a Contract with special abilities for Action. But I can't think of a great replacement name.